src/Form/Element/PhoneNumber.php

```diff
@@ -28,7 +29,7 @@ final class PhoneNumber extends Element implements InputProviderInterface
 {
     /** @inheritDoc */
     protected $attributes = [
-        'type' => 'text',
+        'type' => 'tel',
```

Comment:
That is what I would have expected. Have we overlooked this so far?

Reply:
Yes, it got missed